# Automatically Generated in 2017-06-12 22:33:47.452693
# MESSAGE: HEARTBEAT
# Description:Heartbeat status message
uint8 ID = 1
uint8 sysid
uint8 compid
uint32 timestamp
uint16 battery_voltage
uint8 remote_control
uint8 drive_mode
float32 rc_velocity
float32 rc_steering_front
float32 rc_steering_rear
